src/travel_tracker_service.js

```diff
@@ -69,9 +65,43 @@ async function getTravelerDestinations(travelerTrips, traveler) {
   return { travelerTrips, traveler, travelerDestinations };
 }
 
+
+async function getAllTripsForAgent() {
+  const url =
+    "https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/trips/trips";
+  let response = await fetch(url);
+  let awaitedResponse = await response.json();
+  let trips = awaitedResponse.trips;
+  let allTrips = [];
+  var i;
+  for (i = 0; i < trips.length; i++) {
+    let trip = trips[i];
+    allTrips.push(trip);
+  }
+  return allTrips;
+}
+
+async function getAllDestinationsForAgent() {
+  const url =
+    "https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/destinations/destinations";
+  let response = await fetch(url);
+  let destinationsResponse = await response.json();
+  let destinations = destinationsResponse.destinations;
+  let allDestinationsArr = [];
```

Comment:
why not just call this all destinations?

then you could delete line 95

Reply:
Good call :)